cfpb/consumerfinance.gov
# Update email-signup from DS@0.21.9

Includes https://github.com/cfpb/design-system/pull/1594
Includes https://github.com/cfpb/design-system/pull/1599

## Changes

- Update email-signup from DS@0.21.9
- Move email signup related unit test HTML_SNIPPETS to external mock files.
- Minor adjustment to Summary content gradient.

## How to test this PR

1. PR checks should pass.
2. Pull branch, run local server and `yarn build`
3. http://localhost:8000/language/ar/ should have correct looking email signup in the sidebar.
4. http://localhost:8000/language/ should have correct looking email signup in the sidebar.
5. http://localhost:8000/an-essential-guide-to-building-an-emergency-fund/ should have correct looking email signup (button only)
